src/views/Create.vue

```diff
@@ -153,60 +156,76 @@ watch(selectedNetworkId, () => {
 <template>
   <div>
     <Container class="pt-5">
-      <h2>Deploy space</h2>
-      <div class="s-box pt-4">
-        <SIObject
-          v-model="metadataForm"
-          :error="metadataFormErrors"
-          :definition="metadataDefinition"
-        />
+      <div
+        class="flex fixed top-[72px] inset-x-0 p-3 border-b z-10 bg-skin-bg lg:top-auto lg:inset-x-auto lg:p-0 lg:pr-5 lg:border-0 lg:flex-col lg:translate-x-[-100%] gap-1 min-w-[180px] overflow-auto"
+      >
+        <button
+          v-for="page in pages"
+          ref="pagesRefs"
+          :key="page.id"
+          :disabled="!accessiblePages[page.id]"
+          class="px-3 py-1 block lg:w-full rounded text-left scroll-mr-3 first:ml-auto last:mr-auto"
+          :class="{
+            'bg-skin-active': page.id === currentPage,
+            'hover:bg-skin-hover': page.id !== currentPage
+          }"
+          @click="currentPage = page.id"
+        >
+          {{ page.title }}
+        </button>
       </div>
-
-      <fieldset class="my-3">
-        <legend>Network:</legend>
-
-        <div v-for="network in availableNetworks" :key="network.id">
-          <input v-model="selectedNetworkId" :value="network.id" type="radio" />
-          <label class="ml-2" :for="network.id">{{ network.name }}</label>
-        </div>
-      </fieldset>
-
-      <div class="s-box">
-        <h3 class="mb-2">Settings</h3>
-        <SIObject
-          v-model="settingsForm"
-          :error="settingsFormErrors"
-          :definition="settingsDefinition"
+      <div class="mt-8 lg:mt-0">
+        <BlockSpaceFormProfile
+          v-if="currentPage === 'profile'"
+          :form="metadataForm"
+          @errors="v => handleErrors('profile', v)"
+        />
+        <BlockSpaceFormNetwork v-else-if="currentPage === 'network'" v-model="selectedNetworkId" />
```

Comment:
For consistency and readability we need every prop on new line here

Reply:
It's autoformatted by Prettier.